packages/zowe-explorer/src/uss/utils.ts

```diff
@@ -24,10 +35,12 @@ import { ZoweLogger } from "../utils/LoggerUtils";
 export function injectAdditionalDataToTooltip(node: ZoweUSSNode, tooltip: string): string {
     ZoweLogger.trace("uss.utils.injectAdditionalDataToTooltip called.");
     if (node.downloaded && node.downloadedTime) {
-        // TODO: Add time formatter to localization so we will use not just US variant
-        return `${tooltip} (Downloaded: ${new Date(node.downloadedTime)
-            .toISOString()
-            .replace(/(\d{4})-(\d{2})-(\d{2})T((\d{2}):(\d{2}):([^Z]+))Z/, "$5:$6 $2/$3/$1")})`;
+        const downloadedTime = new Date(node.downloadedTime).toLocaleString(vscode.env.language);
+        tooltip += localize("zowe.uss.utils.tooltip.downloaded", "  \nDownloaded: {0}", downloadedTime);
+        const encodingString = node.binary ? localize("zowe.uss.utils.tooltip.binary", "Binary") : node.encoding;
+        if (encodingString != null) {
+            tooltip += localize("zowe.uss.utils.tooltip.encoding", "  \nEncoding: {0}", encodingString);
```

Comment:
Do you mind hard-coding the two spaces and the newline so that localization packs can provide the translation w/o having to consider formatting?

```suggestion
        tooltip += "  \n" + localize("zowe.uss.utils.tooltip.downloaded", "Downloaded: {0}", downloadedTime);
        const encodingString = node.binary ? localize("zowe.uss.utils.tooltip.binary", "Binary") : node.encoding;
        if (encodingString != null) {
            tooltip += "  \n" + localize("zowe.uss.utils.tooltip.encoding", "Encoding: {0}", encodingString);
```